### PR TITLE
feat: add instance dashboard

### PR DESCRIPTION
**Pull request by cbourgois**
### Description of the Change

40702079 — feat: add instance dashboard

/cc @jleveugle @marie-j

### CODE REVIEW

File: packages/manager/modules/pci/src/projects/project/instances/instance/attach-volume/attach-volume.html [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    <div data-ng-if="$ctrl.volumes.length === 0">
+        <oui-message data-type="warning">
+            <span data-translate="pci_projects_project_instances_instance_attach-volume_error_no_compatible_volume"></span>
+        </oui-message>
+    </div>
+    <div data-ng-if="$ctrl.volumes.length > 0">
+        <oui-select
+            data-name="selectedVolume"
+            data-model="$ctrl.selectedVolume"
+            data-placeholder="Rechercher"
```
Comment (cbourgois) [inline packages/manager/modules/pci/src/projects/project/instances/instance/attach-volume/attach-volume.html]: Should be a translation

File: packages/manager/modules/pci/src/projects/project/instances/instance/attach-volume/attach-volume.routing.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    .state('pci.projects.project.instances.instance.attachVolume', {
+      url: '/attach?storageId',
+      views: {
+        modal: {
+          component: 'pciInstancesInstanceAttachVolume',
+        },
+      },
+      layout: 'modal',
+      resolve: {
+        goBack: /* @ngInject */ ($rootScope, $state, projectId, instance) => () => $state.go('pci.projects.project.instances.instance', {
```
Comment (marie-j) [inline packages/manager/modules/pci/src/projects/project/instances/instance/attach-volume/attach-volume.routing.js]:
```suggestion
        goBack: /* @ngInject */ ($state, projectId, instance) => () => $state.go('pci.projects.project.instances.instance', {
```

File: packages/manager/modules/pci/src/projects/project/instances/instance/instance.html [hunk 3]
```diff
@@ -0,0 +1,224 @@
...
+                            data-translate="pci_projects_project_instances_instance_price_month_value"
+                            data-translate-values="{ price : $ctrl.instancePrice.price.text }"></span>
+                        <span data-ng-if="!$ctrl.instance.isMonthlyBillingEnabled()"
+                            data-translate="pci_projects_project_instances_instance_price_hourly_value"
+                            data-translate-values="{ price : $ctrl.instancePrice.price.text }"></span>
+                    </oui-tile-description>
+
+                    <oui-action-menu data-compact data-align="end"
+                        data-ng-if="!$ctrl.instance.isMonthlyBillingEnabled()">
+                        <oui-action-menu-item text="{{:: 'pci_projects_project_instances_instance_price_action_activeMonthlyBilling_label' | translate }}"
```
Comment (marie-j) [inline packages/manager/modules/pci/src/projects/project/instances/instance/instance.html]: `text` is deprecated

File: packages/manager/modules/pci/src/projects/project/instances/instance/instance.html [hunk 4]
```diff
@@ -0,0 +1,224 @@
...
+                        <span data-ng-bind="volume.size * 1024 * 1024 * 1024 | cucBytes:2:'true'"></span>
+                    </oui-tile-description>
+                    <oui-tile-description
+                        data-ng-if="$ctrl.instance.volumes.length === 0">
+                        <span>-</span>
+                    </oui-tile-description>
+
+
+                    <oui-action-menu data-compact data-align="end">
+                        <oui-action-menu-item text="{{:: 'pci_projects_project_instances_instance_volumes_action_manage_label' | translate }}"
```
Comment (marie-j) [inline packages/manager/modules/pci/src/projects/project/instances/instance/instance.html]: Same here

File: packages/manager/modules/pci/src/projects/project/instances/instance/vnc/vnc.html [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+</oui-message>
+
+<div class="embed-responsive embed-responsive-16by9 h-50"
+    data-ng-if="$ctrl.vncInfos.url">
+    <iframe class="embed-responsive-item" data-ng-src="{{ $ctrl.vncInfos.url }}" seamless cuc-autofocus></iframe>
+</div>
+
+<a data-ng-href="{{ $ctrl.vncInfos.url }}"
+    class="oui-link_icon"
+    target="_blank">
```
Comment (marie-j) [inline packages/manager/modules/pci/src/projects/project/instances/instance/vnc/vnc.html]:
```suggestion
    target="_blank"
    rel="noopener">
```

**Pull request merged**